content/en/tracing/trace_collection/custom_instrumentation/android/otel.md

```diff
@@ -0,0 +1,514 @@
+        }
+    }
+   ```
+   {{% /tab %}}
+   {{% tab "Java" %}}
+   ```java
+   public class SampleApplication extends Application {
+        @Override
+        public void onCreate() {
+            super.onCreate();
+            Configuration configuration =
+                    new Configuration.Builder(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>)
+                            .useSite(DatadogSite.US1_FED)
+                            .build();
+            Datadog.initialize(this, configuration, trackingConsent);
+        }
+    }
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+   {{< /site-region >}}
+
+   {{< site-region region="ap1" >}}
+   {{< tabs >}}
+   {{% tab "Kotlin" %}}
+   ```kotlin
+   class SampleApplication : Application() {
+        override fun onCreate() {
+            super.onCreate()
+            val configuration = Configuration.Builder(
+                    clientToken = <CLIENT_TOKEN>,
+                    env = <ENV_NAME>,
+                    variant = <APP_VARIANT_NAME>
+                )
+                .useSite(DatadogSite.AP1)
+                .build()
+            Datadog.initialize(this, configuration, trackingConsent)
+        }
+    }
+   ```
+   {{% /tab %}}
+   {{% tab "Java" %}}
+   ```java
+   public class SampleApplication extends Application {
+        @Override
+        public void onCreate() {
+            super.onCreate();
+            Configuration configuration =
+                    new Configuration.Builder(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>)
+                            .useSite(DatadogSite.AP1)
+                            .build();
+            Datadog.initialize(this, configuration, trackingConsent);
+        }
+    }
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+   {{< /site-region >}}
+
+   To be compliant with the GDPR regulation, the SDK requires the tracking consent value at initialization.
+   The tracking consent can be one of the following values [See Tracking Consent][6].:
+   * `TrackingConsent.PENDING`: The SDK starts collecting and batching the data but does not send it to the data
+     collection endpoint. The SDK waits for the new tracking consent value to decide what to do with the batched data.
+   * `TrackingConsent.GRANTED`: The SDK starts collecting the data and sends it to the data collection endpoint.
+   * `TrackingConsent.NOT_GRANTED`: The SDK does not collect any data. You will not be able to manually send any logs, traces, or
+     RUM events.
+
+   To update the tracking consent after the SDK is initialized, call: `Datadog.setTrackingConsent(<NEW CONSENT>)`.
+   The SDK changes its behavior according to the new consent. For example, if the current tracking consent is `TrackingConsent.PENDING` and you update it to:
+   * `TrackingConsent.GRANTED`: The SDK sends all current batched data and future data directly to the data collection endpoint.
+   * `TrackingConsent.NOT_GRANTED`: The SDK wipes all batched data and does not collect any future data.
```

Comment:
```suggestion
   To be compliant with the GDPR regulation, the SDK requires the tracking consent value at initialization.
   For more information, see [Tracking Consent][6].
```
I don't think we should duplicate the information that already exists in the Tracking Consent section.